oneview_redfish_toolkit/api/computer_system.py

```diff
@@ -109,3 +113,59 @@ def map_boot(self, boot_list):
             redfish_boot_list.append('None')
 
         return redfish_boot_list
+
+    def get_oneview_power_configuration(self, reset_type):
+        """Maps Redfish's power options to OneView's power option
+
+            Maps the known Redfish power options to OneView Power option.
+            If a unknown power option shows up it will raise an Exception.
+
+            Args:
+                reset_type: Redfish power option.
+
+            Returns:
+                dict: Dict with OneView power configuration.
+
+            Exception:
+                OneViewRedfishError: raises an exception if
+                reset_type is an unmapped value.
+        """
+
+        reset_type_dict = dict()
```

Comment:
Can we build this dictionary statically and load it only once instead of each time this method is called?

Reply:
Maybe a good solution would be I create a singleton class (ComputerSystemPower) to power.

Reply:
Fixed.